core/trino-main/src/main/resources/webapp/src/components/LivePlan.jsx

```diff
@@ -18,14 +18,7 @@ import ReactDOMServer from "react-dom/server";
 import * as dagreD3 from "dagre-d3";
 import * as d3 from "d3";
 
-import {
-    formatRows,
-    getStageStateColor,
-    initializeGraph,
-    initializeSvg,
-    parseAndFormatDataSize,
-    truncateString
-} from "../utils";
+import {formatRows, getStageStateColor, initializeGraph, initializeSvg, parseAndFormatDataSize, truncateString} from "../utils";
```

Comment:
Unrelated change. Revert

Reply:
The reason that these .jsx files are changed is that the import format is changed, which will not affect the semantic meaning of these classes, I would revert these changes though.